lib/winapi/Makefile

```diff
@@ -1,4 +1,5 @@
 include $(NXDK_DIR)/lib/winapi/winmm/Makefile
+include $(NXDK_DIR)/lib/winapi/ws2_32/Makefile
```

Comment:
ws2_32 should be split from this PR, cleaned, polished, and upstreamed.

Maybe @dracc or @Teufelchen1 (also @thrimbor or @ryzee119, but those got too many tasks already)